test/PoolManagerInitialize.t.sol

```diff
@@ -316,10 +317,10 @@ contract PoolManagerInitializeTest is Test, Deployers, GasSnapshot {
         );
         manager.initialize(uninitializedKey, sqrtPriceX96, ZERO_BYTES);
         // protocol fees should default to 0
-        (Pool.Slot0 memory slot0,,,) = manager.pools(uninitializedKey.toId());
-        assertEq(slot0.protocolFee, 0);
+        (,, uint16 slot0ProtocolFee,) = manager.getSlot0(key.toId());
```

Comment:
did you mean to replace `uninitializedKey` with `key` here?

Reply:
yess thanks